### PR TITLE
pls merge

### PR DESCRIPTION
**Pull request by rcrcrcrcrcrcrc**
check for null

### CODE REVIEW

File: commands/moderation/say.js [hunk 1]
```diff
@@ -7,14 +7,18 @@ module.exports = {
     usage: "<input>",
     run: (client, message, args) => {
         message.delete();
-        message.channel.send(args.join(" "));
+        if(args[0])
```
Comment (Henessly) [inline commands/moderation/say.js:10]: Wouldn't you want to check the length of the array incase of an out of bounds exception. That's how I would tackle this in java but idfk how js arrays work.

like `args.length() > 0`

File: commands/moderation/say.js [hunk 2]
```diff
@@ -7,14 +7,18 @@ module.exports = {
...
+            message.channel.send(args.join(" "));
+        message.channel.send("Cannot send an empty message.");
```
Comment (Henessly) [inline commands/moderation/say.js:12]: Lets say our if statement passes as true and it sends the parsed message. What stops the program from running this line? There's no return in the if statement so both messages will be sent correct?

**Review by Henessly — COMMENTED** [review]
Just some quick comments.

**Comment by Henessly** [comment]
This is gonna get merged either way but I'm just interested in that command.

**Comment by Henessly** [comment]
The plan was to edit his commit then merge lol.

**Comment by axissnova** [comment]
yolo

**Pull request merged**